fineract-provider/src/main/resources/db/changelog/tenant/parts/OXY-44_add_isAccountLevelArrearsToleranceEnable_in_loan_product_table.xml

```diff
@@ -24,11 +24,11 @@
 
         <preConditions onFail="MARK_RAN">
             <not>
-                <columnExists tableName="m_product_loan" columnName="is_account_level_arrears_tolerance_enable"/>
+                <columnExists tableName="m_product_loan" columnName="is_account_level_arrears_tolerance_enable_1"/>
```

Comment:
I think the "_1" you added should be on the migration name not column name in precondition